webapp/channels/src/components/more_direct_channels/list_item/list_item.tsx

```diff
@@ -75,9 +75,12 @@ const ListItem = React.forwardRef((props: Props, ref?: React.Ref<HTMLDivElement>
             }
 
             <div className='more-modal__actions'>
-                <div className='more-modal__actions--round'>
+                <button
+                    className='more-modal__actions--round'
+                    aria-label={`Add option ${option.label}`}
```

Comment:
@hmhealey can you please confirm if we should be translating the aria-label strings too?

Reply:
Yeah, they should be translated as well. Since this component is functional, you can use `useIntl` to get an `intl` object and then call `intl.formatMessage` to do the translating